connection_go18.go

```diff
@@ -160,6 +161,11 @@ func (mc *mysqlConn) watchCancel(ctx context.Context) error {
 	select {
 	default:
 	case <-ctx.Done():
+		ctxErr := ctx.Err()
+		killErr := mc.kill()
+		if killErr != nil {
```

Comment:
IMHO, it is not need to return kill error to client, it will be enough to log this error, since it is not fatal, the server will close query in any time.

Reply:
`errLog.Printf("failed to kill query %s", killErr)`

Reply:
ok